pkg/common/peertracker/tracker_linux.go

```diff
@@ -30,7 +30,7 @@ type linuxWatcher struct {
 	gid       uint32
 	pid       int32
 	mtx       sync.Mutex
-	procfh    *os.File
+	procfh    int
```

Comment:
nit: `procfd` seems more appropriate now?

Reply:
SGTM. Naming! I need to push another commit anyway with one more small change.